### PR TITLE
Feature/bar specs

### PR DESCRIPTION
**Pull request by DerekMaffett**
Adds some basic specs to cover the `Bar` primitive.

### CODE REVIEW

File: test/client/spec/victory-primitives/bar.spec.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    const barShape = SvgTestHelper.getBarShape(wrapper);
+
+    expect(barShape.width).to.eql(10);
+  });
+
+  it("should apply a default width", () => {
+    const wrapper = shallow(<Bar {...baseProps}/>);
+    const barShape = SvgTestHelper.getBarShape(wrapper);
+
+    expect(barShape.width).to.eql(0.5);
```
Comment (boygirl) [inline test/client/spec/victory-primitives/bar.spec.js]: This default width seems arbitrary. Can you include an explanation or calculation of the default width (from padding and data length)

**Comment by boygirl** [comment]
Approved pending improvement to default width spec.

File: test/client/spec/victory-primitives/bar.spec.js (hunk 1 above)
Comment (DerekMaffett) [reply test/client/spec/victory-primitives/bar.spec.js]: Agreed. Thanks for calling that out.

**Review by boygirl — APPROVED** [review]
Thanks!

**Pull request merged**